src/app/qgsprojectproperties.cpp

```diff
@@ -154,6 +154,28 @@ QgsProjectProperties::QgsProjectProperties( QgsMapCanvas *mapCanvas, QWidget *pa
   updateGuiForMapUnits( QgsProject::instance()->crs().mapUnits() );
   projectionSelector->setCrs( QgsProject::instance()->crs() );
 
+  QPolygonF mainCanvasPoly = mapCanvas->mapSettings().visiblePolygon();
+  QgsGeometry g = QgsGeometry::fromQPolygonF( mainCanvasPoly );
+  // close polygon
+  mainCanvasPoly << mainCanvasPoly.at( 0 );
+  if ( mapCanvas->mapSettings().destinationCrs() !=
```

Comment:
Probably safest to use QgsProject::instance()->crs() here, although generally it should be the same as the canvas crs